package.json

```diff
@@ -47,6 +48,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
     "prebuild": "prebuildify -t 8.14.0 -t electron@3.0.0 --napi --strip",
+    "prebuild:cross": "prebuildify-cross",
```

Comment:
So this would be invoked e.g. as:

```
PLATFORM=linux ARCH=arm ARM_VERSION=7 npm run prebuild:cross
```

Reply:
easy as: https://github.com/ahdinosaur/prebuildify-cross/commit/5cba6e01d37fee7298c1f496a6380a7f1c58301a